tests/core/test_variables.py

```diff
@@ -466,34 +456,6 @@ def test_call__end_attribute__formulas__different_names():
     assert simulation.calculate('end_attribute__formulas__different_names', month) == 300
 
 
-def test_clone__end_attribute__formulas__different_names():
```

Comment:
Why did you choose to delete the clone testing ?

Reply:
Because this test was testing the cloning of a `Formula`, which is an object that does not exist anymore.